src/Core__Date.resi

```diff
@@ -65,10 +191,27 @@ external makeWithYMDHMSM: (
   ~milliseconds: int,
 ) => t = "Date"
 module UTC: {
-  @val external makeWithYM: (~year: int, ~month: int) => time = "Date.UTC"
-  @val external makeWithYMD: (~year: int, ~month: int, ~date: int) => time = "Date.UTC"
+  /** 
+  Same as Date.makeWithYM, but returns the time since 1 January 1970 UTC in milliseconds.
```

Comment:
Could we add examples here too, preferably also comparing this with it's "regular" equivalent function?

And maybe a link to somewhere you can read about UTC vs regular dates? If such a thing exists on MDN.